jobs/fxci-taskcluster-export/fxci_etl/loaders/bigquery.py

```diff
@@ -17,13 +17,16 @@
 
 @dataclass
 class Record(ABC):
+    submission_date: str
     table_name: InitVar[str]
 
     def __post_init__(self, table_name):
         self.table = table_name
 
     @classmethod
     def from_dict(cls, table_name: str, data: dict[str, Any]) -> "Record":
+        current_date = datetime.now().date()
```

Comment:
```suggestion
        current_date = datetime.utcnow().date()
```

Reply:
Fyi `datetime.utcnow()` is deprecated, but your point still stands (should be `datetime.now(timezone.utc)`